libethereum/Executive.cpp

```diff
@@ -66,9 +67,15 @@ void StandardTrace::operator()(uint64_t _steps, Instruction inst, bigint newMemS
 
 	Json::Value r(Json::objectValue);
 
+	if (m_lastCallData != ext.data.toBytes())
```

Comment:
This comparison is quite expensive. Perhaps it is better to check whether the call depth increased (see below)?